versions/1/elasticsearch/spec.yml

```diff
@@ -0,0 +1,28 @@
+spec:
+  additionalContents: false
+  contents:
+    - description: Folder containing ML model assets
+      type: folder
+      name: ml_model
+      required: false
+      contents:
+        - description: An ML model file
+          type: file
+          contentMediaType: "application/json"
+          pattern: '^{PACKAGE_NAME}-.+\.json$'
+    - description: Folder containing Elasticsearch Ingest Node pipeline definitions
```

Comment:
Do you need the ingest pipeline to be installed as unrelated from data stream?

Reply:
Yes - that's why I've added the top-level ingest pipeline here. Part of the changes to the Fleet code for this include installing pipelines that are not part of a data stream.